test/unit/fake/initialization.spec.ts

```diff
@@ -3,10 +3,12 @@ import { makeRandomAddress } from '@src/utils';
 import { Receiver, Receiver__factory, Returner } from '@typechained';
 import receiverArtifact from 'artifacts/test/contracts/watchable-function-logic/Receiver.sol/Receiver.json';
 import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
 import { ethers, network } from 'hardhat';
 import storageArtifact from 'test/unit/fake/testdata/Storage.json';
 
 chai.use(smock.matchers);
+chai.use(chaiAsPromised);
```

Comment:
ey @5h4z4mm, why do we need both here?

Reply:
@wei3erHase Good call, we don't need the `smock.matchers` plugin here. Fixed now.